### PR TITLE
[FAB-109] HiFi Lobby

### PR DESCRIPTION
**Pull request by daystram**
- [x] Lobby
- [x] Waiting room
- [x] Complete story
- [x] bug: double checkpoint place on pointer leave for fill tool

### CODE REVIEW

**Comment by daystram** [comment]
Will wait for #41 to merge before merging this

File: fableous-fe/src/containers/HubCanvasPage.tsx
Comment (nicolauscg) [inline fableous-fe/src/containers/HubCanvasPage.tsx:395]: do we want to make this and one below it xs={12} and either sm={6} or md={6} ?

**Review by nicolauscg — COMMENTED** [review]
all good!

in hub page, change isAllControllersJoined func back to `.every(...)` if its unintentional

File: fableous-fe/src/containers/HubCanvasPage.tsx
Comment (daystram) [reply fableous-fe/src/containers/HubCanvasPage.tsx]: done

**Review by nicolauscg — APPROVED** [review]

**Pull request merged**
